joint_names: [joint1, joint2, joint3, joint4, joint5, joint6]

joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

joint_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints: 
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
  required_drive_mode: 8
  constraints:
    stopped_velocity_tolerance: 20
    joint1: {trajectory: 50, goal: 50}
    joint2: {trajectory: 50, goal: 50}
    joint3: {trajectory: 50, goal: 50}
    joint4: {trajectory: 50, goal: 50}
    joint5: {trajectory: 50, goal: 50}
    joint6: {trajectory: 50, goal: 50}

